examples/tv-casting-app/tv-casting-common/include/CastingServer.h

```diff
@@ -450,7 +450,7 @@ class CastingServer
     uint16_t mTargetVideoPlayerDeviceType                                 = 0;
     char mTargetVideoPlayerDeviceName[chip::Dnssd::kMaxDeviceNameLen + 1] = {};
     char mTargetVideoPlayerHostName[chip::Dnssd::kHostNameMaxLength + 1]  = {};
-    size_t mTargetVideoPlayerNumIPs                                       = 0; // number of valid IP addresses
+    uint8_t mTargetVideoPlayerNumIPs                                      = 0; // number of valid IP addresses
```

Comment:
It might be cleaner going forward if we matched the data type of this to what is used elsewhere in the SDK: https://github.com/project-chip/connectedhomeip/blob/master/src/lib/dnssd/Resolver.h#L46